src/modules/common/actions/burnToken.ts

```diff
@@ -0,0 +1,86 @@
+import { DispatchRequest, getQuery, RequestAction } from '@redux-requests/core';
+import { createAction as createSmartAction } from 'redux-smart-actions';
+import { RootState } from 'store';
+import { Store } from 'redux';
+import { setAccount } from 'modules/account/store/actions/setAccount';
+import {
+  BounceErc721,
+  BounceErc1155,
+} from '../../web3/contracts';
+import { NftType } from 'modules/createNFT/actions/createNft';
+import { TransactionReceipt } from '@ethersproject/abstract-provider';
+
+export const burnToken = createSmartAction(
```

Comment:
```suggestion
export const burnToken = createSmartAction<RequestAction<Data, TransformedData>>(
```
I think need to write typings for that.

Reply:
This action is working with the smart contract, it didn't have any type. Maybe we can put <RequestAction<void, void>>, I think we also can ignore it.

Reply:
```
<RequestAction<void, void>>
```

void, void is ok